resources/translations/fi.edn

```diff
@@ -168,15 +168,15 @@
          :apply "Hae"
          :apply-for-bundle "Hae %1 aineistoa yhdessä "
          :header "%1 aineistoa korissa"
-         :intro [:div.intro [:p "Lisää resursseja koriin hakeaksesi niitä. Samalla hakemuksella haettavat resurssit ryhmitellään automaattisesti yhteen."]]
+         :intro [:div.intro [:p "Hae käyttöoikeuksia lisäämällä resursseja koriin. Useampaa oikeutta kerralla haettaessa keskenään samaa hakemusmenettelyä noudattavat resurssit yhdistyvät yhdeksi hakemukseksi."]]
          :remove "Poista"}
   :catalogue {:catalogue "Aineistoluettelo"
               :header "Aineistoluettelo"
-              :intro [:p "Aineistoluettelossa on listattuna kaikki resurssit joita voit tästä järjestelmästä hakea."]
+              :intro [:p "Allaolevien resurssien käyttöoikeudet ovat haettavissa."]
               :more-info "Lisätietoja"
               :continue-existing-application "Jatka hakemusta"
-              :continue-existing-application-intro [:div.intro [:p "Sinulla on keskeneräisiä hakemuksia, joita voit jatkaa tästä:"]]
-              :apply-resources "Hae uusia resursseja"}
+              :continue-existing-application-intro [:div.intro [:p "Keskeneräiset hakemukset:"]]
```

Comment:
I really like this kind of compact expression, I think it's a definite improvement over the old one.

Reply:
It could even be "Keskeneräiset hakemuksesi:"

Reply:
"Keskeneräiset hakemuksesi:" is fine. I generally try to avoid too much "you" in the texts but it can be there.

Reply:
Opted to not add the possessive suffix for consistency reasons: it's not used elsewhere in the translations either (e.g., "Jatka hakemusta").